ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -50,12 +48,17 @@ export default class StatusDropdown extends React.Component {
         this.props.alertSummary,
         this.props.frameworks,
       ),
+      isWeekend: this.isWeekend(),
     };
-
-    this.isWeekend = false;
-    this.showCountdownToTriageIcon = true;
   }
 
+  isWeekend = () => {
+    const currentDate = new Date(Date.now());
+    const currentDay = currentDate.getDay();
+
+    return currentDay === weekdays.saturday || currentDay === weekdays.sunday;
+  };
+
```

Comment:
This method can be moved in the helper file (`alertCountdownHelper.js`) and also be used on `ui/perfherder/alerts/AlertStatusCountdown.jsx` line 44.